easybuild/easyconfigs/s/Spyder/Spyder-6.0.1-gfbf-2023b.eb

```
@@ -0,0 +1,118 @@
+easyblock = 'PythonBundle'
+
+name = 'Spyder'
+version = '6.0.1'
+
+homepage = 'https://github.com/spyder-ide/spyder'
+description = """Spyder is an interactive Python development environment providing MATLAB-like features in a simple
+ and light-weighted software."""
+
+toolchain = {'name': 'gfbf', 'version': '2023b'}
+
+builddependencies = [
+    ('poetry', '1.6.1'),
+    ('maturin', '1.3.1'),
+]
+dependencies = [
+    ('Python', '3.11.5'),
+    ('Python-bundle-PyPI', '2023.10'),
+    ('aiohttp', '3.9.5'),
+    ('IPython', '8.17.2'),
+    ('jedi', '0.19.1'),
+    ('jupyter-server', '2.14.0'),
+    ('Pylint', '3.2.5'),
+    ('PyZMQ', '25.1.2'),
+    ('PyQt5', '5.15.10'),
+    ('Qtconsole', '5.6.0'),
+    ('Rtree', '1.2.0'),
+    ('Deprecated', '1.2.14'),
+    ('tornado', '6.4'),
+    ('numpydoc', '1.8.0'),
+    ('cookiecutter', '2.6.0'),
+    ('wurlitzer', '3.1.1'),
+    ('python-lsp-server', '1.12.0'),
+]
+
+use_pip = True
+
+# fix deleting of PYTHONPATH
+local_preinstallopts = "sed -i '13,19d' spyder/app/start.py && "
+local_preinstallopts += "sed -i '171,173d' spyder/plugins/ipythonconsole/utils/kernelspec.py && "
+
+exts_list = [
+    ('asyncssh', '2.17.0', {
+        'checksums': ['3b159c105aa388c1e2245c4faf483f540ada8cad99402281119100166e5edb3c'],
+    }),
+    ('diff-match-patch', '20230430', {
+        'checksums': ['953019cdb9c9d2c9e47b5b12bcff3cf4746fc4598eb406076fa1fc27e6a1f15c'],
+    }),
+    ('importlib_metadata', '8.4.0', {
+        'checksums': ['9a547d3bc3608b025f93d403fdd1aae741c24fbb8314df4b155675742ce303c5'],
+    }),
+    ('pygithub', '2.4.0', {
```

I'd suggest the "official" spelling:
```suggestion
    ('PyGithub', '2.4.0', {
        'source_tmpl': SOURCELOWER_TAR_GZ,
```